src/qml/pages/onboarding/Onboarding00Welcome.qml

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// The Onboarding00Welcome page.
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+Column {
+    width: appTitle.width
+    spacing: 16
+
+    Image {
+        id: appLogo
+        anchors.horizontalCenter: parent.horizontalCenter
+        source: "image://images/app"
+        sourceSize.width: 128
+        sourceSize.height: 128
+    }
+
+    Text {
```

Comment:
21c969272e718e63d3de00a9fe9e0f702eb4fd9f

Should use `Label` instead.

Reply:
`Label` extends `Text`. Which `Label`'s properties we are interested in?

Reply:
I’d say background and palette.

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui-qml/pull/43#issuecomment-933918510).